Assets/Scripts/Framework/TargetSelector.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Framework
+{
+    public class TargetSelector : MonoBehaviour
+    {   
+        [SerializeField] private List<Transform> targetList;
```

Comment:
```suggestion
        [SerializeField] private List<Transform> routePoints;
```
Het woord `target` is meer iets als parameter in een functie. Dit is gewoon een varible in dit script.